pkg/consts/consts.go

```diff
@@ -23,15 +23,14 @@ const (
 	// ProtoIndexWrapperTypeID is included in each encoded IndexWrapper to help prevent
 	// decoding binaries that are not actually IndexWrappers.
 	ProtoIndexWrapperTypeID = "INDX"
+
+	// NamespaceSize is the size of a namespace ID in bytes.
+	NamespaceSize = 32
```

Comment:
should this be 33?

Reply:
I plan on adding the version byte in a follow up PR. At that point, I think it possible this constant remains 32 because I'm thinking of having something like this [playground](https://go.dev/play/p/cGtQby1ChR8) where app/core have a Namespace struct that contains a version (1 byte) and ID (32 bytes). 

We would call `.Bytes()` on the Namespace struct prior to pushing to the NMT so the NMT only deals with 33 byte leaves.